BSP/STM32F413/Src/BSP_Contactor.c

```diff
@@ -1,64 +1,105 @@
 /* Copyright (c) 2018-2022 UT Longhorn Racing Solar */
 
 #include "BSP_Contactor.h"
-#include "stm32f4xx.h"
 
 /**
- * @brief   Initializes the GPIO pins that interfaces with the Contactor.
- *          Two GPIO pins are initialized. One as an output and one as an input.
- *          The output pin controls the state and the input pin views what state the contactor is through the
- *          Aux pins.
- * @param   None
- * @return  None
+ * @brief Initializes all Contactor pins used by the BPS
```

Comment:
Make contactors and Fans application level libraries. Since they both use the BSP for PWM and are applied for their own specific peripheral.